File: Coding Problems & My Solutions/Fizz Buzz/Fizz_Buzz.java
class Solution {
    public List<String> fizzBuzz(int n) {
        List<String> arrayList = new ArrayList<String>();
        for ( int i = 1; i <= n; i++ ){
            String s = "";
            if (i % 3== 0){
                s = s+"Fizz";
            }
            if (i % 5 == 0){
                s = s+"Buzz";
            }
            if (s.equals("")){
                arrayList.add(""+i);
            }
            else {
                arrayList.add(""+s);
            }
        }
        return arrayList;
    }
}

File: Coding Problems & My Solutions/Fizz Buzz/Fizz_Buzz.swift
class Solution {
    func fizzBuzz(_ n: Int) -> [String] {
        var list = [String]()
        for num in 1...n {
            switch num {
                case let num where num % 3 == 0 && num % 5 == 0:
                    list.append("FizzBuzz")
                case let num where num % 3 == 0:
                    list.append("Fizz")
                case let num where num % 5 == 0:
                    list.append("Buzz")
                default:
                    list.append(String(num))
            }
        }
        return list
    }
}